glib/src/value.rs

```diff
@@ -454,6 +462,42 @@ impl Uninitialized for Value {
     }
 }
 
+impl ToValue for Value {
```

Comment:
```suggestion
impl<'a> ToValue for &'a Value {
```

Would (also) be useful to have. Can you add that?